x/evm/types/msg_confirm_gateway_txs.go

```diff
@@ -0,0 +1,69 @@
+package types
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	"github.com/axelarnetwork/axelar-core/utils"
+	nexus "github.com/axelarnetwork/axelar-core/x/nexus/exported"
+	"github.com/axelarnetwork/utils/slices"
+)
+
+const TxLimit = 100
+
+// NewConfirmGatewayTxsRequest creates a message of type ConfirmGatewayTxsRequest
+func NewConfirmGatewayTxsRequest(sender sdk.AccAddress, chain string, txIDs []Hash) *ConfirmGatewayTxsRequest {
+	return &ConfirmGatewayTxsRequest{
+		Sender: sender,
+		Chain:  nexus.ChainName(utils.NormalizeString(chain)),
+		TxIDs:  txIDs,
+	}
+}
+
+// Route implements sdk.Msg
+func (m ConfirmGatewayTxsRequest) Route() string {
+	return RouterKey
+}
+
+// Type implements sdk.Msg
+func (m ConfirmGatewayTxsRequest) Type() string {
+	return "ConfirmGatewayTx"
+}
+
+// ValidateBasic implements sdk.Msg
+func (m ConfirmGatewayTxsRequest) ValidateBasic() error {
+	if err := sdk.VerifyAddressFormat(m.Sender); err != nil {
+		return sdkerrors.Wrap(sdkerrors.ErrInvalidAddress, sdkerrors.Wrap(err, "sender").Error())
+	}
+
+	if err := m.Chain.Validate(); err != nil {
+		return sdkerrors.Wrap(err, "invalid chain")
+	}
+
+	if len(m.TxIDs) > TxLimit {
+		return fmt.Errorf("txIDs limit exceeded")
```

Comment:
```suggestion
		return fmt.Errorf("tx ids limit exceeded")
```
the error message below does not do camel case so keep it consistent.